chart/values.yaml

```diff
@@ -28,18 +76,15 @@ monitoring:
   enabled: false
 
 mongodb:
-  enabled: false
+  enabled: true
+  nameOverride: datasets-server-mongodb
```

Comment:
Yes, it is used by the mongo chart to prefix the name of the chart.
We have to do this to avoid conflict with others mongodb deployed in the same stack ;)